protocol_driver_test_fixture.cc

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 Google LLC
```

Comment:
Should be 2022

Reply:
Done

Reply:
Have to run clang-format on lot of files. Let us do that outside current pull request ?